### PR TITLE
PHOENIX-7246: Add support for mapping JDBC type code 0 to protobuf NULL type

### PR DESCRIPTION
**Pull request by satyakommula96**

### CODE REVIEW

**Comment by satyakommula96** [comment]
please review this PR @stoty

File: python-phoenixdb/phoenixdb/types.py [hunk 1]
```diff
@@ -298,6 +298,8 @@ def from_column(column):
 
     @staticmethod
     def _from_jdbc(jdbc_code):
+        if jdbc_code == 0:
```
Comment (stoty) [inline python-phoenixdb/phoenixdb/types.py:301]: This is a bit ugly.
Would it be possible to add this to the type maps instead ?

Comment (satyakommula96) [reply python-phoenixdb/phoenixdb/types.py]: addressed

File: python-phoenixdb/phoenixdb/types.py [hunk 2]
```diff
@@ -222,9 +225,9 @@ def __cmp__(self, other):
     (2011, common_pb2.STRING),  # NCLOB
     (2009, common_pb2.STRING),  # SQLXML
     # Returned by Avatica for Arrays in EMPTY resultsets
-    (2000, common_pb2.BYTE_STRING)  # JAVA_OBJECT
+    (2000, common_pb2.BYTE_STRING),  # JAVA_OBJECT
     # These are defined by JDBC, but cannot be mapped
-    # NULL
+    (0, common_pb2.NULL) # NULL
```
Comment (stoty) [inline python-phoenixdb/phoenixdb/types.py:230]: The comment is obviously not true for this one.
Please move this to the start (in numerical order)

Comment (stoty) [reply python-phoenixdb/phoenixdb/types.py:230]: They are not numerically ordered.
Still, it should be grouped with the other standard Phoenix types.

**Comment by stoty** [comment]
Committed with minor cleanups.

Thanks a lot for the patch:

Some nits for the future:
- We do not use colon after the JIRA ID in the commit message
- The commit message should match the JIRA description
- Run flake8 / pylint on the code, it is discussed in the README

**Pull request closed without merging**